uui/components/overlays/Alert.module.scss

```diff
@@ -10,11 +10,22 @@
 }
 
 .alert-wrapper {
-    padding: 12px 18px 12px 12px;
+    padding: 12px 18px 9px 9px;
     display: flex;
+    align-items: center;
     overflow: hidden;
     min-width: 170px;
     border-radius: var(--uui-alert-border-radius);
+    height: 48px;
+}
+
+.alert-wrapper-compact {
```

Comment:
We don't need to duplicate styles which isn't different between sizes. Let's leave wrapper with common styles and create 2 more classes: size-48, size-36 which will have only related for them styles.

Reply:
Done